src/main/resources/model/instance.json

```diff
@@ -220,36 +220,46 @@
       "type": "object",
       "properties": {
         "id": {
+          "searchAliases": "item.id",
           "index": "keyword"
         },
         "hrid": {
+          "searchAliases": "item.hrid",
           "index": "keyword"
         },
         "accessionNumber": {
+          "searchAliases": "item.accessionNumber",
           "index": "source"
         },
         "formerIds": {
+          "searchAliases": "item.formerIds",
           "index": "source"
         },
         "itemIdentifier": {
+          "searchAliases": "item.itemIdentifier",
           "index": "source"
         },
         "barcode": {
+          "searchAliases": "item.barcode",
           "index": "keyword"
         },
         "effectiveLocationId": {
+          "searchAliases": "item.effectiveLocationId",
           "searchTypes": [ "facet", "filter" ],
           "index": "keyword"
         },
         "statisticalCodeIds": {
+          "searchAliases": "item.statisticalCodeIds",
           "searchTypes": [ "facet", "filter" ],
           "index": "keyword"
         },
         "status": {
+          "searchAliases": "item.status",
           "type": "object",
           "properties": {
             "name": {
               "searchTypes": [ "facet", "filter", "sort" ],
+              "searchAliases": "item.status.name",
               "index": "keyword",
               "sort": {
                 "fieldName": "items.status.name",
```

Comment:
It's no way to add searchAliases or second fieldName for sort parameter.
So for now I added `sort_item.status.name` field, and Processor for this filed that should be removed with renaming items
```
 "sort_item.status.name": {
      "searchTypes": "sort",
      "type": "search",
      "index": "lowercase_sort",
      "processor": "sortItemStatusProcessor"
    },
```